__init__.py

```diff
@@ -507,27 +491,29 @@ def handle_set_alarm(self, message):
         self.enclosure.activate_mouth_events()
 
     def _get_alarm_name(self, utt):
-        """ Get the alarm name using regex on an utterance
-        """
+        """Get the alarm name using regex on an utterance."""
         self.log.debug("Utterance being searched: " + utt)
+        invalid_names = self.translate_list('invalid_names')
         rx_file = self.find_resource('name.rx', 'regex')
         if utt and rx_file:
+            patterns = []
             with open(rx_file) as f:
-                for pat in f.read().splitlines():
-                    pat = pat.strip()
-                    self.log.debug("Regex pattern: " + pat)
-                    if pat and pat[0] == "#":
-                        continue
-                    res = re.search(pat, utt)
-                    if res:
-                        try:
-                            name = res.group("Name").strip()
-                            self.log.debug('Regex name extracted: '
-                                           + name)
-                            if name and len(name.strip()) > 0:
-                                return name.lower()
-                        except IndexError:
-                            pass
+                pass
```

Comment:
What does `pass` do in this context?
I thought it would only be needed if nothing was being done in this block.

Reply:
Seems like a piece of debugging code that weren't removed, will fix.